Styleguide.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Salmon Cookies Style Guide</title>
+    <link rel="stylesheet" href="stylesheet.css" media="screen" title="no title" charset="utf-8">
+  </head>
+  <header>
+    <h2>Header Section</h2>
+      <code>Font: Georgia;
+        Font-size: 12px;
+        <div id = 'headColor'>
+          Background-color: #17A5FF;
+        </div>
+      </code>
+    <h2>Logo</h2>
+      <code>
+        Image: large salmon;
+        Height: 100px;
+        Width: 100px;
+      </code>
+  </header>
+  <body>
+    <section id = "allH1">
+      <h2>H1</h2>
+      <code>Font-size: 20px;</code>
+    </section>
+    <section id = "allH2">
+      <h2>H2</h2>
+      <code>Font size: 15px;</code>
+    </section>
+    <section id = 'allH3'>
+      <h2>H3</h2>
+      <code>Font-size: 10px;</code>
+    </section>
+    <section id = 'customFont'>
+      <h2>Custom Font</h2>
+      <p>Cookie (Custom Google Font)<link href='https://fonts.googleapis.com/css?family=Cookie' rel='stylesheet' type='text/css'></p>
+    </section>
+    <section id = 'standardFont'>
+      <h2>Standard Font - Data</h2>
+      <code>Font: Georgia
+      <div id = 'standardColor'>
+        Color: Black
+      </div>
+      </code>
+      <h2>Standard Font - Text</h2>
+      <code>Font: Verdana</code>
+      <div id = 'standard2'>Color: #191200</div>
+    </section>
+    <section id = 'links'>
+      <h2>Link Font</h2>
+      <div id = 'linkColor'>Color: #318ADE</div>
+      <h2>Visited Link Font</h2>
+      <div id = 'visitedColor'>Color: #00912D</div>
+      <h2>Mouse Over Link</h2>
+      <div id = 'mouseColor'>Color: #b21f0e</div>
+    </section>
+    <section id = 'backgroundStuff'>
+      <h2>Background Image</h2>
+      <code>None</code>
+      <h2>Background Color</h2>
+      <div id = 'backgroundColorBlock'>#197D8F</div>
+      <!--(for my reference later).gradient {
+      background-image:
+      linear-gradient(
+      #197D8F, #28C9E5
+      );
+    }-->
+    </section>
+    <section id = 'buttons'>
+      <h2>Buttons</h2>
+      <p>To be determined...</p>
+    </section>
+
+  </body>
+  <footer id = "footer">
+  	<p>Copyright by Toonces</p>
+  	<code>
+  		Font: Georgia
+  		Font-size: 10px
+	  </code>
+    <div id ='footColor'>Color: #003B5E</div>
+  </footer>
+</html>

```

Comment:
Remove any white space between your =
Also, you can use <span> for text instead of <code> (on lines 79 and 82). More conventional that way.